site/ko/swift/python_interoperability.ipynb

```diff
@@ -0,0 +1,687 @@
+        "#@title Licensed under the Apache License, Version 2.0 (the \"License\"); { display-mode: \"form\" }\n",
+        "// Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+        "// you may not use this file except in compliance with the License.\n",
+        "// You may obtain a copy of the License at\n",
+        "//\n",
+        "// https://www.apache.org/licenses/LICENSE-2.0\n",
+        "//\n",
+        "// Unless required by applicable law or agreed to in writing, software\n",
+        "// distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "// See the License for the specific language governing permissions and\n",
+        "// limitations under the License."
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "sI1ZtrdiA4aY"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/swift/tutorials/python_interoperability\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/ko/swift/python_interoperability.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/ko/swift/python_interoperability.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "8sa42_NblqRE"
+      },
+      "source": [
+        "# 파이썬 상호 호환성\n",
+        "\n",
+        "텐서플로를 위한 스위프트는 파이썬 상호 호환성을 지원합니다.\n",
+        "스위프트에서 파이썬 모듈을 임포트해서, 스위프트와 파이썬 사이의 값을 바꾸거나 파이썬 함수를 호출할 수 있습니다."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "kZRlD4utdPuX",
+        "outputId": "67d314d6-413f-4c65-cba3-600b650e62b4",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 51
+        }
+      },
+      "source": [
+        "import Python\n",
+        "print(Python.version)"
+      ],
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "3.6.8 (default, Jan 14 2019, 11:02:34) \r\n",
+            "[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]\r\n"
+          ],
+          "name": "stdout"
+        }
```

Comment:
Almost every code section are executed. Original document does not have any output results. 
Please check rest of code whether output exists.

Reply:
Thanks for pointing out an important issue.

I have imported the original file and moved the translation to fix the problem.